cmd/notation/inspect.go

```diff
@@ -235,18 +248,41 @@ func getSignedAttributes(outputFormat string, envContent *signature.EnvelopeCont
 	return signedAttributes
 }
 
-func getUnsignedAttributes(envContent *signature.EnvelopeContent) map[string]string {
-	unsignedAttributes := map[string]string{}
+func getUnsignedAttributes(outputFormat string, envContent *signature.EnvelopeContent) (map[string]any, error) {
```

Comment:
If unsigned attributes can be unstable, having errors in parsing unsigned attributes should not stop the inspect process. How about adding an `error` field to `timestampSignature`?

For example:
```
myRegistry/myRepo@sha256:c0669ef34cdc14332c0f1ab0c2c01acb91d96014b172f1a76f3a39e63d1f0bda
└── application/vnd.cncf.notary.signature
    └── sha256:20171244b2291f7e854e595a20842faa581d82ebcd85266543390d5ca4b73551
        ├── media type: application/cose
        ├── signature algorithm: RSASSA-PSS-SHA-256
        ├── signed attributes
        │   ├── signingScheme: notary.x509
        │   └── signingTime: Fri Jul 26 14:50:29 2024
        ├── user defined attributes
        │   └── (empty)
        ├── unsigned attributes
        │   ├── timestampSignature
        │   │   └── error: failed to parse timestamp countersignature
```

Reply:
That's a good idea.

Reply:
Updated.